tests/load/test_sql_client.py

```diff
@@ -295,7 +300,7 @@ def test_commit_transaction(client: SqlJobClientBase) -> None:
 
 @pytest.mark.parametrize('client', ALL_CLIENTS, indirect=True)
 def test_rollback_transaction(client: SqlJobClientBase) -> None:
```

Comment:
this won't work on Athena

Reply:
I have amended the caps with a setting wether transactions are supported